kommunicate/src/main/java/io/kommunicate/KmConversationBuilder.java

```diff
@@ -250,4 +251,14 @@ public KmConversationBuilder setTeamId(String teamId) {
         this.teamId = teamId;
         return this;
     }
+
+    public void updateConversationInfo(String key, String value) {
+        if (conversationInfo != null) {
```

Comment:
how about simplify it further to make it in 3 lines

if (conversationInfo == null) {
          conversationInfo = new HashMap<>();
}
conversationInfo.put(key, value);